File: lc001-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> res;
        int i,j;
        for (i=0;i<=nums.size()-2;i++){
            for (j=i+1;j<=nums.size()-1;j++){
                if(nums[i]+nums[j]==target){
                    res.push_back(i+1);
                    res.push_back(j+1);
                    break;
                }
            }
        }
        return res;
    }
};
//res.push_back是从后面push进来

File: lc001-1.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        res = []
        n = len(nums)
        for i in range(n):
            for j in range(i):
                if(nums[i] + nums[j] == target):
                    res = [i, j]
                    break
        return res
